src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowSkeleton.java

```diff
@@ -3,6 +3,7 @@
 import com.minecolonies.api.util.Log;
 import com.minecolonies.blockout.controls.Button;
 import com.minecolonies.blockout.controls.ButtonHandler;
+import com.minecolonies.blockout.controls.Label;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.blockout.controls.Label'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)